packages/lesswrong/themes/themeType.ts

```diff
@@ -0,0 +1,405 @@
+// eslint-disable-next-line no-restricted-imports
+import type { Color as MuiColorShades } from '@material-ui/core';
+import type { PartialDeep, Merge } from 'type-fest'
+
+declare global {
+  type BreakpointName = "xs"|"sm"|"md"|"lg"|"xl"|"tiny"
+  type ColorString = string;
+  
+  type ThemeGreyscale = MuiColorShades & {
+    0: ColorString,
+    1000: ColorString,
+    
+    10: ColorString,
+    20: ColorString,
+    25: ColorString,
+    30: ColorString,
+    40: ColorString,
+    55: ColorString,
+    60: ColorString,
+    110: ColorString,
+    120: ColorString,
+    140: ColorString,
+    250: ColorString,
+    310: ColorString,
+    315: ColorString,
+    320: ColorString,
+    340: ColorString,
+    410: ColorString,
+    550: ColorString,
+    620: ColorString,
+    650: ColorString,
+    680: ColorString,
+  }
+  type ThemeShadePalette = {
```

Comment:
```suggestion
  type ThemeBasePalette = {
```
?

ShadePalette is already a historical name.